PSix/FBRequest.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  FBRequest.swift
+//  PSix
+//
+//  Created by Turzion, Avihu on 4/20/15.
+//  Copyright (c) 2015 PayPal. All rights reserved.
+//
+
+import Foundation
+import Parse
+
+
+typealias FBParamsList = [NSObject: AnyObject]
+
+let URI_SEP = "/"
+
+
+class FBRequest {
+    
+    private(set) var path: String
```

Comment:
I want to make available for subclasses as well when replicating the path. I hate this, but I'm not sure what's the best way to do it. `private` hides it from subclasses when I want to copy it.